crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -2560,3 +2559,24 @@ mod test {
         );
     }
 }
+
+#[derive(Serialize, Deserialize, Debug, Copy, Clone, Deserializable, PartialEq, Eq, ValueEnum)]
+#[serde(rename_all = "camelCase")]
+pub enum UIMode {
```

Comment:
Nit: Is there a reason for hoisting this to the CLI module? It feels like it should live in the `turbo_json` or `config` layer. Not that we're in a place to split up `turborepo-lib`, but if we did I think `cli` would still be an "entry" module that depends on `config`/`turbo_json` modules.

Also if we're moving the implementation, we should move the unit tests for ser/de with it.